CODEOWNERS

```diff
@@ -1,7 +1,7 @@
 # SPDX-License-Identifier: Apache-2.0
 
-# FireFly Core Maintainers
-* @peterbroadhurst @nguyer @awrichar
+# FireFly transaction manager maintainers
+* @hyperledger/firefly-transaction-manager-maintainers
 
 # FireFly Documentation Maintainers
-/docs @peterbroadhurst @nguyer @awrichar @nickgaski
\ No newline at end of file
+/docs @peterbroadhurst @nguyer @awrichar @nickgaski
```

Comment:
I think we can drop this section. We started with the idea of separate docs maintainers early on, but in practice have never used that idea.